utils/build-script

```diff
@@ -1319,6 +1319,9 @@ def main_normal():
         toolchain.libtool = args.host_libtool
     if args.cmake is not None:
         toolchain.cmake = args.cmake
+    if args.sccache:
+        print("Ensuring the sccache server is running...")
+        shell.capture([toolchain.sccache, "--show-stats"], stderr=shell.DEVNULL)
```

Comment:
(optional) Could we add a comment on why we prefer `--show-stats` to `--start-server`?
(optional) Also, would it be possible to preserve `stderr`? That would help investigating issues